.circleci/config.yml

```diff
@@ -17,25 +17,31 @@ jobs:
           name: Linter
           command: python3 -m black --check ./
   pytest:
-    machine: true
+    machine:
+      image: ubuntu-2004:202201-02
+      docker_layer_caching: true
     steps:
       - checkout
       - run:
-          name: "Build and Run all containers"
+          name: "Docker Compose Build"
           command: |
-            env DOCKER_BUILDKIT=0 docker-compose --file docker/docker-compose.pytest.yml up --build --abort-on-container-exit
-  build:
-    machine: true
-    steps:
-      - checkout
-      # build the application image
+            docker-compose --file docker/docker-compose.pytest.yml build
       - run:
-          name: Docker Build
-          command: docker build -f docker/services/web/Dockerfile -t $DOCKER_REGISTRY_NAME/climatemindbackend:$CIRCLE_BUILD_NUM .
+          name: "Docker Compose Up"
+          command: docker-compose --file docker/docker-compose.pytest.yml up -d
+      - run:
+          name: "Wait for database to spin up"
+          command: sleep 30
+      - run:
+          name: "Run Pytest"
+          command: docker exec -it docker-web-1 pytest --cipdb
+          # or to NOT check for breakpoints accidentally left in the code
+          # docker-compose exec -it docker-web-1 pytest -n auto
   cypress:
     # Set up test environment and run cypress tests
     machine:
       image: ubuntu-2004:202201-02
+      docker_layer_caching: true
```

Comment:
This doesn't work for cypress for some reason, it was faster before. So I suggest to fix or remove

Reply:
I think there was just a weird issue with Cypress that day. It was sitting there for a while doing nothing as the clock was ticking up.